data-model/src/test/java/org/pdxfinder/graph/dao/DataProjectionsTimeTest.java

```diff
@@ -0,0 +1,87 @@
+package org.pdxfinder.graph.dao;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.pdxfinder.BaseTestWithPersistence;
+import org.pdxfinder.graph.repositories.DataProjectionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+public class DataProjectionsTimeTest extends BaseTestWithPersistence {
+
+    private static final String LABEL = "TEST";
+
+    @Autowired
+    private DataProjectionRepository dataProjectionRepository;
+
+    @Ignore
+    @Test
+    public void Given_DataProjection_When_CascadeCreateSaveAndFinds_Then_PrintTimes(){
+
+        byte[] hundredBytes = generateRandomByte(2);
+        byte[] tenKiB = generateRandomByte(4);
+        byte[] hundredKib = generateRandomByte(5);
+        byte[] oneMib = generateRandomByte(6);
+        byte[] tenMiB = generateRandomByte(7);
+
+        //to init db. Else the first time test is off by a second.
+        timeCreateSaveAndFindAndReturnElapsedTime(hundredBytes);
+
+        List<Long> elapsedTime = timeCreateSaveAndFindAndReturnElapsedTime(tenKiB);
+        List<Long> elapsedTime1 = timeCreateSaveAndFindAndReturnElapsedTime(hundredKib);
+        List<Long> elapsedTime2 = timeCreateSaveAndFindAndReturnElapsedTime(oneMib);
+        List<Long> elapsedTime3 = timeCreateSaveAndFindAndReturnElapsedTime(tenMiB);
+
+
+        String timeTable = String.format(" Data Size: %s Time %s %s %s Average %s \n", tenKiB.length, elapsedTime.get(0), elapsedTime.get(1), elapsedTime.get(2), elapsedTime.get(3)) +
+                String.format(" Data Size: %s Time %s %s %s Average %s \n", hundredKib.length, elapsedTime1.get(0), elapsedTime1.get(1), elapsedTime1.get(2), elapsedTime1.get(3)) +
+                String.format(" Data Size: %s Time %s %s %s  Average %s \n", oneMib.length, elapsedTime2.get(0), elapsedTime2.get(1), elapsedTime2.get(2), elapsedTime2.get(3)) +
+                String.format(" Data Size: %s Time %s %s %s Average %s \n", tenMiB.length, elapsedTime3.get(0), elapsedTime3.get(1), elapsedTime3.get(2), elapsedTime3.get(3));
+        System.out.println(timeTable);
+    }
+
+
+    private byte[] generateRandomByte(int exponent){
+
+        int byteSize = (int) Math.pow(10, exponent);
+        byte[] generatedByte = new byte[byteSize];
+        new Random().nextBytes(generatedByte);
+        return generatedByte;
+    }
+
+    private List<Long> timeCreateSaveAndFindAndReturnElapsedTime(byte[] randomData){
```

Comment:
Extract the byte array to string conversion.  As is, this test couples the time taken to generate the test data with the time it takes to save it (i.e., it will naturally multiply the time taken as java will have to allocate and process a larger string which will, of course, take more time).  Generate the random string outside the test and pass that to the method rather than passing in the byte array.